### ISSUE TITLE
Make Contributor Landing Page Link More Prominent

### ISSUE THREAD

**Issue by PixelantDesign** [gitcoinco/Marketing#7]
As a contributor it's hard for me to see the contributor landing page link.

### Definition of Done

Update the Green bar area and call out the contributor landing page a little more.

<img width="1403" alt="screen shot 2018-09-20 at 9 57 33 am" src="https://user-images.githubusercontent.com/238235/45823474-c718d600-bcbb-11e8-920c-24cef5ebb7d6.png">

**Comment by willsputra** [comment]
Here's an idea! :)

![landing](https://user-images.githubusercontent.com/19514207/46298894-c4c74d80-c5d2-11e8-99a9-b8b646434f22.jpg)

**Comment by owocki** [comment]
I really like this!

**Comment by willsputra** [comment]
@owocki thanks!
@PixelantDesign thoughts?

**Comment by PixelantDesign** [comment]
@willsputra looks good. Could we bring the callout and buttons closer together?

**Comment by willsputra** [comment]
@PixelantDesign this?

![landing](https://user-images.githubusercontent.com/19514207/46882808-92dfa200-ce82-11e8-86fb-79000b8e3cbc.jpg)

**Comment by thelostone-mc** [comment]
Is this ready ? 
I'd like to pass this onto the community to see if anyone is interested in picking this up !  ^_^

**Comment by owocki** [comment]
yes can we pls have a 'code this' ticket?  i will happily bounty it

**Comment by Anish-Agnihotri** [comment]
Hey! I'm current working on [#2282](https://github.com/gitcoinco/web/issues/2282) and [#2283](https://github.com/gitcoinco/web/issues/2283) which are somewhat focused around this. 

I'd be glad to incorporate this in as I make the new CTA in the same PR if you'd like.

**Comment by thelostone-mc** [comment]
@Anish-Agnihotri have at it ^_^

**Comment by Anish-Agnihotri** [comment]
Perfect, thanks! Looking at an ETA of under 48 hours, but I'll keep you updated.

**Comment by thelostone-mc** [comment]
If there was a diff design mock up for the options you'd listed out, I would be open to it but currently with what we have I'd stick the [green banner](https://github.com/gitcoinco/Marketing/issues/7#issuecomment-429417052)

When @PixelantDesign originally designed it -> this landing page was intended for funder and contributors would navigate to their own landing page @ `/contibutor`

I'm open to it but I believe the scope of this issue is being over complicated
